setup.py

```diff
@@ -17,12 +17,12 @@
 from setuptools import find_packages, setup
 
 setup(
-    name='interface',
-    version='0.1.0',
+    name="interface",
```

Comment:
is there a reason why double quotes are used here?

Reply:
No specific reason, this is the result of linting using the `black` formatter.

Reply:
I seee. I just noticed that `setup.py` isn't included in the `lint` subroutine in Makefile, can you please make that change?

Reply:
Yup, the change has been made.